src/App.tsx

```diff
@@ -5,7 +5,11 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const onAdd = (movie: Movie) => setMovies([...movies, movie]);
```

Comment:
Better use prevState for such logic
```suggestion
  const onAdd = (movie: Movie) => setMovies(prevMovies => ([...prevMovies, movie]));
```